api/src/indigo_object.cpp

```diff
@@ -122,8 +122,6 @@ IndigoObjectTypes::IndigoObjectTypes()
     emplace(IndigoObject::TGROUP, "TGroup");
     emplace(IndigoObject::TGROUPS_ITER, "TGroupsIterator");
     emplace(IndigoObject::GROSS_REACTION, "GrossReaction");
-    emplace(IndigoObject::JSON_LOADER, "JsonLoader");
-    emplace(IndigoObject::JSON_SAVER, "JsonSaver");

```

Comment:
If you are removing it here, also remove corresponding IndigoObject items

Reply:
https://github.com/epam/Indigo/pull/224#discussion_r571992182

Reply:
Fixed